Cargo.lock

```diff
@@ -689,16 +689,6 @@ dependencies = [
  "syn 1.0.85",
 ]
 
-[[package]]
```

Comment:
So this removes a duplicated "dash-map" but upgrade us to tokio 1.17 and introduces a duplication of tokio-util because a lot of other packages don't have updates there either.  @bartlomieju thoughts?

Reply:
tower-lsp 0.16.0 requires upgrade of the tokio and the libc 😢

Reply:
Yes, it isn't a criticism... just want @bartlomieju thoughts on it.

Reply:
I think it's okay. I can upgrade these dependencies in a separate PR before landing this okay. I'll get back to you once this is done.

Reply:
Okay I did update the dependencies. @CGQAQ please rebase against `main`